servlet-feature-pack/src/main/resources/modules/system/layers/base/org/wildfly/extension/undertow/main/module.xml

```diff
@@ -51,7 +51,7 @@
         <module name="org.jboss.staxmapper"/>
         <module name="org.jboss.as.clustering.common"/>
         <module name="org.wildfly.clustering.service"/>
-        <module name="org.wildfly.clustering.web.api" optional="true"/>
+        <module name="org.wildfly.clustering.web.container"/>
```

Comment:
org.wildfly.clustering.web.container is a new module being introduced by this PR.
org.wildfly.clustering.web.api is actually an obsolete module dependency.

Reply:
Can it not be optional? This now makes it a requirement which means it can't be stripped down.

Reply:
No, it cannot.  The org.wildfly.clustering.web.container module only contains a set of interfaces - which were refactored from the undertow subsystem itself.  This module is what allows the org.wildfly.clustering.web.extension module (containing the distributable web subsystem which configures the distributed session manager implementations) to avoid a direct dependency on the undertow subsystem, while allowing the undertow subsystem to avoid a direct dependency on any clustering implementation module.